ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -17,6 +17,8 @@ function populate_artifact_dir() {
   case "${CLUSTER_TYPE}" in
     aws|aws-arm64)
       grep -Po 'Instance ID: \Ki\-\w+' "${dir}/.openshift_install.log" > "${SHARED_DIR}/aws-instance-ids.txt";;
+    alibabacloud)
+      awk -F'id=' '/alicloud_instance.*Creation complete/{ print $2 }' "${dir}/.openshift_install.log" | tr -d ']"' > "${SHARED_DIR}/alibaba-instance-ids.txt";;
```

Comment:
We could add outputs from the terraform for the instance IDs to make it easier to collect them.